### DISCUSSION TITLE
About assignment problem 1.2..

### DISCUSSION THREAD

**Discussion by hichoe95** [seominjoon/kaist-ai605#21] [Q&A] [answered]
I think when Q and K are vectors, then the problem holds, maybe.
Is it right?

**Comment by seominjoon** [comment, marked as answer]
Could you please clarify what you mean by "problem holds"?

**Reply by hichoe95** [reply]
I'm sorry about using unclear expression.
It means ' The 'Var(QK^T/sqrt(d_k)) = 1' makes sense when Q and K are vectors.'
In other words, I wondered what the variance of matrix means. Is it the variance of each element of the matrix? ( i.e. variance of each Q_i K_i^T, i is the number of rows)

But I think 'question #24' may be the solution to it.

Thank you!

**Reply by seominjoon** [reply]
I see where you were confused. This is actually also discussed at #24. The notation was a bit lazy but what I meant is that every element of whatever goes in Var(), which is a matrix, is 1.

**Reply by hichoe95** [reply]
Thank you for your answer.!
